src/sub-blocks/PriceCard/PriceCard.scss

```diff
@@ -0,0 +1,121 @@
+@import '../../../styles/variables.scss';
+@import '../../../styles/mixins';
+
+$block: '.#{$ns}price-card';
+
+#{$block} {
+    position: relative;
+
+    &__bg {
+        position: absolute;
+        top: 0;
+        left: 0;
+        right: 0;
+        bottom: 0;
+    }
```

Comment:
what is it for? we don't use image for background. can we set color to the main cards-component?

Reply:
Replaced div with `BackgroundImage` component according to src/sub-blocks/BackgroundCard/BackgroundCard.tsx and renamed this `&__bg` to `&__background`